src/main/java/net/snowflake/spark/snowflake/s3upload/ExecutorServiceResultsHandler.java

```diff
@@ -84,7 +84,6 @@ public void abort() {
             executorService.shutdownNow();
         }
         executorService = null;
-        completionService = null;
```

Comment:
Why do we still need to remove this line if we move streamManager.complete() out of finally{}?

Reply:
You are correct, either moving the `streamManager.complete()` out of the finally block or removing the code that nulls out the `completionService` will fix the `NullPointerException`.  I did both to make `ExecutorServiceResultsHandler` safer in general to prevent `NullPointerException`s.  However, since not-nulling out `completionService` is not required I'm fine reverting the change.

Reply:
Thanks.